keycloak-iam/src/main/java/com/valtimo/keycloak/service/RequestScopeUserCache.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.valtimo.keycloak.service
+
+import com.ritense.valtimo.contract.authentication.ManageableUser
+import com.ritense.valtimo.contract.authentication.model.ValtimoUser
+import mu.KotlinLogging
+import kotlin.reflect.KClass
+
+class RequestScopeUserCache(
```

Comment:
Can you please make this implement an interface?

Reply:
It does now